<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorControls</class>
 <widget class="QWidget" name="ColorControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="16" column="1">
    <widget class="QSpinBox" name="exposureSpinBox">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QSlider" name="exposureSlider">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="Line" name="separatorLine1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QSlider" name="saturationSlider">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="hueSpinBox">
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="brightnessLabel">
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="saturationSpinBox">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="colorLabel">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QSpinBox" name="brightnessSpinBox">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="bnwLabel">
     <property name="text">
      <string>Black &amp; White</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="hueLabel">
     <property name="text">
      <string>Hue</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="Line" name="separatorLine2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lightLabel">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Light</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="saturationLabel">
     <property name="text">
      <string>Saturation</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="contrastLabel">
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QSlider" name="brightnessSlider">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="bnwButton">
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="tintLabel">
     <property name="text">
      <string>Tint</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="applyColorButton">
     <property name="text">
      <string>Apply color changes</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="tintSpinBox">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="exposureLabel">
     <property name="text">
      <string>Exposure</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSlider" name="hueSlider">
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="invertLabel">
     <property name="text">
      <string>Invert</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QSlider" name="contrastSlider">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QSlider" name="tintSlider">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="invertButton">
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QSpinBox" name="contrastSpinBox">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QPushButton" name="applyLightingButton">
     <property name="text">
      <string>Apply lighting changes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tintSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tintSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightnessSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrastSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrastSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tintSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tintSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightnessSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightnessSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrastSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrastSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
